contracts/RemixRewardUpgradable.sol

```diff
@@ -129,7 +136,15 @@ contract Remix is Initializable, ERC721Upgradeable, ERC721EnumerableUpgradeable,
         tokensData[tokenId].hash = contributorHash;
     }
 
-    function grantRemixersMinting (address[] calldata remixers, uint amount) public onlyRole(DEFAULT_ADMIN_ROLE)  {
+    function addTrainer (address trainer) public onlyRole(DEFAULT_ADMIN_ROLE) {
+        trainers[trainer] = 1;
+    }
+
+    function removeTrainer (address trainer) public onlyRole(DEFAULT_ADMIN_ROLE) {
+        delete trainers[trainer];
+    }
+
+    function grantRemixersMinting (address[] calldata remixers, uint amount) public isTrainer()  {
```

Comment:
With this update, admin will not be able to call this method. Is that intentional?

Reply:
yes I did that to make the function as light as possible. An admin can just add himself as trainer.